### PR TITLE
[styles] Add generic craft=* type to mapcss-mapping.csv 

### PR DESCRIPTION
**Pull request by dvdmrtnz**
We have a generic craft icon and we have a style defined in `Icons.mapcss`. However, we did not have a type defined in `mapcss-mapping.csv`, so the generic craft POIs were never showing. This PR fixes it.

Also the consensus in https://github.com/organicmaps/organicmaps/issues/5950 seems to be that using the hand icon rotated is better for the generic craft types so I rotated it. 

* Fixes https://github.com/organicmaps/organicmaps/issues/5950
* Related to https://github.com/organicmaps/organicmaps/issues/3867

---

### CODE REVIEW

**Review by biodranik — COMMENTED** [review]
Introducing generic type doesn't sound right, but may be useful in some cases. @vng WDYT?

**Comment by vng** [comment]
Well, we have a generic shop, so there is no problem with craft, since there are a lot (in sum) minor crafts that we definitely can't support.

File: data/categories.txt [hunk 1]
```diff
@@ -10203,6 +10203,44 @@ sk:Mobilný operátor
...
+ru:Мастерская
+sk:Remeselník
+sv:Hantverk
+sw:Ufundi
+th:งานฝีมือ
+tr:Zanaat
+uk:Майстерня
+vi:Thủ công
+zh-Hans:工艺作坊
+zh-Hant:工藝作坊
```
Comment (vng) [inline data/categories.txt:10242]: Well, should either add ```@craft``` category into every craft-* entry or remove this generic term.

I prefer to remove it now.

**Review by vng — CHANGES_REQUESTED** [review]

File: data/categories.txt (hunk 1 above)
Comment (dvdmrtnz) [reply data/categories.txt]: Removed from `categories.txt`. 

`[type.craft]` was already in `types_strings.txt`.

**Pull request merged**